components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.v1/src/main/java/org/wso2/carbon/identity/api/server/userstore/v1/core/ServerUserStoreService.java

```diff
@@ -1020,4 +1022,70 @@ private String extractPropertyName(String pathProp) {
         }
         return name;
     }
+
+    /**
+     * Get user store attributes mappings for a given user store type id.
+     *
+     * @param typeId String user store type id.
+     * @return UserStoreAttributeMapping user store attribute mappings.
+     */
+    public UserStoreAttributeMapping getUserStoreMappingAttributes(String typeId) {
+
+        Set<String> classNames;
+        String userStoreName = getUserStoreType(base64URLDecodeId(typeId));
+        if (StringUtils.isBlank(userStoreName)) {
+            throw handleException(Response.Status.BAD_REQUEST, UserStoreConstants.ErrorMessage.
+                    ERROR_CODE_INVALID_INPUT);
+        }
+        try {
+            UserStoreConfigService userStoreConfigService = UserStoreConfigServiceHolder.getInstance()
+                    .getUserStoreConfigService();
+            classNames = userStoreConfigService.getAvailableUserStoreClasses();
+            if (CollectionUtils.isEmpty(classNames) ||
+                    !classNames.contains(userStoreName)) {
```

Comment:
Formatting issue. this condition could be moved to the above line

Reply:
fixed in https://github.com/wso2/identity-api-server/pull/307